### PR TITLE
chore(cli): add comments on app template Cargo.toml

### PR DESCRIPTION
**Pull request by lucasfernog**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: tooling/cli/templates/app/src-tauri/Cargo.crate-manifest [hunk 1]
```diff
@@ -20,5 +20,9 @@ serde = { version = "1.0", features = ["derive"] }
 tauri = {{{  tauri_dep  }}}
 
 [features]
+# by default Tauri runs on production mode
```
Comment (JonasKruckenberg) [inline tooling/cli/templates/app/src-tauri/Cargo.crate-manifest]: Although (I just learned) both "in production mode" and "on production mode" would be fine (I think) "in" sounds way nicer to me 🤔 
example "you have to run this application in compatibility mode"

**Review by FabianLars — DISMISSED** [review]

File: tooling/cli/templates/app/src-tauri/Cargo.crate-manifest (hunk 1 above)
Comment (FabianLars) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest]: feels like that's because we're german lol

Comment (JonasKruckenberg) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest]: riiiight, but I actually googled this 😂 It's not just my brain liking it more, the internet is on my side here 😂

File: tooling/cli/templates/app/src-tauri/Cargo.crate-manifest [hunk 2]
```diff
@@ -20,7 +20,7 @@ serde = { version = "1.0", features = ["derive"] }
 tauri = {{{  tauri_dep  }}}
 
 [features]
-# by default Tauri runs on production mode
+# by default Tauri runs in production mode
 # when `tauri dev` runs it is executed with `cargo run --no-default-features` if `devPath` is an URL
 default = [ "custom-protocol" ]
 # this feature is used on production builds and when `devPath` points to the filesystem
```
Comment (JonasKruckenberg) [inline tooling/cli/templates/app/src-tauri/Cargo.crate-manifest:26]:
```suggestion
# this feature is used for production builds where `devPath` points to the filesystem
```

Comment (JonasKruckenberg) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest:26]: ahh too late 🤷🏻‍♂️

Comment (lucasfernog) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest:26]: oh nice change, pls PR that one :D

Comment (JonasKruckenberg) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest:26]: I will 👍🏻

Comment (JonasKruckenberg) [reply tooling/cli/templates/app/src-tauri/Cargo.crate-manifest:26]: #3444

**Pull request merged**